lib/scanner.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const objectScan = require('object-scan');
+
+// Traverses object and returns object containing all matches of form:
+// { [needle]: [match, ...] } where match is [value, parents]
+// Refer to object-scan for needle syntax
```

Comment:
Would you be ok with removing this comment entirely? I mean it's very obvious from how it's used and comment always get outdated (I comment very little these days and mostly on things that are not easily deduced from looking at the code).

Reply:
Yes, it can be removed. The interesting part is possible `needle` syntaxes, and those shouldn't be documented here.

Reply:
done